CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 ## vNext (TBD)
 
 ### Enhancements
-* None
+* Added `ClientResetError.backupFilePath` where the backup copy of the realm will be placed once the client reset process has completed ([#1291](https://github.com/realm/realm-dart/pull/1291)).
+* Added `CompensatingWriteError` containing detailed error information about the writes that have been reverted by the server due to permissions or subscription view restrictions. The `Configuration.flexibleSync.syncErrorHandler` will be invoked with this error type when this error occurs ([#1291](https://github.com/realm/realm-dart/pull/1291)).
+* Added `SyncError.detailedMessage` that contains error details. In case of a server error, it contains a link to the server logs with more error details ([#1291](https://github.com/realm/realm-dart/pull/1291)).
+* `SyncError.create` is deprecated and it will be removed. The sync errors will be created only by the internal `realmCore` ([#1291](https://github.com/realm/realm-dart/pull/1291)).
```

Comment:
```suggestion
* `SyncError.create` is deprecated and it will be removed. The sync errors will be created only by internally ([#1291](https://github.com/realm/realm-dart/pull/1291)).
```